web/_data/footer.yml

```diff
@@ -22,4 +22,4 @@ social:
     url: https://www.twitter.com/datagouvfr
   -
     icon: github
-    url: https://github.com/etalab
+    url: https://github.com/etalab/schema.data.gouv.fr
```

Comment:
Je n'ai pas trouvé comment utiliser les variables Jekyll à cet endroit. Apparemment il faut un plugin supplémentaire ([source](https://talk.jekyllrb.com/t/how-do-you-make-jekyll-execute-liquid-variables-in-front-matter/1638))

Reply:
@abulte, un avis ?

Reply:
pas bien grave de laisser en l'état je pense